tests/nix_tests/test_precompiles.py

```diff
@@ -6,13 +6,12 @@
 from .utils import ADDRS, get_precompile_contract, wait_for_fn
 
 
-@pytest.fixture(scope="module", params=[False])
-def ibc(request, tmp_path_factory):
+@pytest.fixture(scope="module")
+def ibc(tmp_path_factory):
     "prepare-network"
```

Comment:
These are docstrings right?
```suggestion
    """
    Prepares the network.
    """
```